liblangutil/CharStream.h

```diff
@@ -97,9 +98,19 @@ class CharStream
 	/// Functions that help pretty-printing parse errors
 	/// Do only use in error cases, they are quite expensive.
 	std::string lineAtPosition(int _position) const;
+
+	/// @returns (1-based) line and column that matches to the byte offset @p _position.
 	std::tuple<int, int> translatePositionToLineColumn(int _position) const;
 	///@}
 
+	/// Translates a line:column to the absolute position.
```

Comment:
Is this also 1-based for both line and column?

Reply:
This is actually documented one line below. I probably just pushed this up and you checked earlier. Sorry :)